ext/span.c

```diff
@@ -861,6 +863,13 @@ void ddtrace_close_span(ddtrace_span_data *span) {
 
         zend_execute_data *execute_data = EG(current_execute_data);
         ddtrace_maybe_add_code_origin_information(span, execute_data && EX(func) && !ZEND_USER_CODE(EX(func)->type));
+
+        if (get_DD_TRACE_RESOURCE_RENAMING_ENABLED() && get_DD_TRACE_SIDECAR_TRACE_SENDER()) {
+            // the purpose is client-computed stats for /v0.6/stats, so sidecar is required
```

Comment:
Um, in case we don't manually compute, the agent will do that task and should?
Also we don't ever drop that tag when computing - Let's always include the tag, if only for consistency.
```suggestion
        if (get_DD_TRACE_RESOURCE_RENAMING_ENABLED()) {
```